argocd/utils.go

```diff
@@ -97,7 +97,7 @@ func isValidPolicyAction(action string) bool {
 		"*":                       true,
 	}
 	validActionPatterns := []*regexp.Regexp{
-		regexp.MustCompile("action/.*"),
+		regexp.MustCompile("(action|update|delete)/.*"),
```

Comment:
Can we use the form which is available in upstream Argo CD?
```suggestion
		regexp.MustCompile("action/.*"),
		regexp.MustCompile("update/.*"),
		regexp.MustCompile("delete/.*"),
```

See PR https://github.com/argoproj/argo-cd/pull/19338/files

<img width="1266" alt="grafik" src="https://github.com/user-attachments/assets/1235594b-a705-46e5-87ad-d9f60884419e" />

Reply:
Hi @mkilchhofer, no problem at all, and thanks for the review!